modules/rosetta/src/main/scala/org/tessellation/rosetta/http/routes/ConstructionRoutes.scala

```diff
@@ -0,0 +1,35 @@
+package org.tessellation.rosetta.http.routes
+
+import cats.effect.Async
+
+import org.tessellation.rosetta.domain.api.construction.ConstructionDerive
+import org.tessellation.rosetta.domain.construction.ConstructionService
+import org.tessellation.rosetta.ext.http4s.refined._
+import org.tessellation.sdk.config.AppEnvironment
+
+import org.http4s.HttpRoutes
+import org.http4s.circe.CirceEntityCodec._
+import org.http4s.dsl.Http4sDsl
+import org.http4s.server.Router
+
+final case class ConstructionRoutes[F[_]: Async](
```

Comment:
I know that we already have routes as `case classes` but I think they shouldn't be `case classes` but regular classes or anonymous traits instead. `case class` implies some immutable data, but this is a service class with dependencies.